extract-initramfs-from-vmlinuz.sh

```diff
@@ -71,34 +71,33 @@ trap "rm -rf ${tmp}" EXIT
 tmp_dec="${tmp}/decompress"
 mkdir "${tmp_dec}"
 fr_prefix="${tmp}/first-round"
-try_unzstd_decompress "${image}" "${tmp_dec}" "${fr_prefix}"
 
-shopt -s failglob
+perform_round() {
+    local image="${1}"
+    local tmp_dec="${2}"
+    local round_prefix="${3}"
+    try_unzstd_decompress "${image}" "${tmp_dec}" "${round_prefix}"
+    for rnd in "${round_prefix}"*; do
+        if [[ $(file --brief "${rnd}") =~ 'cpio archive' ]]; then
+            mkdir -p "${out}/rootfs-${rootfs_idx}"
+            while cpio --quiet --extract --make-directories --directory="${out}/rootfs-${rootfs_idx}" --nonmatching 'dev/*'; do
+                rootfs_idx=$(( rootfs_idx + 1 ))
+                mkdir -p "${out}/rootfs-${rootfs_idx}"
+            done <${rnd}
+            rmdir "${out}/rootfs-${rootfs_idx}"
+        fi
+    done
+}
 
+shopt -s nullglob
```

Comment:
arm64 vmlinuz only require a single round, so the glob at line 80 fails in the second round when `failglob` is set, but works with `nullglob`.